libraries/stdlib/js/src/kotlin/dom/Dom.kt

```diff
@@ -18,3 +21,41 @@ public val Node.isText: Boolean
  */
 public val Node.isElement: Boolean
     get() = nodeType == Node.ELEMENT_NODE
+
+
+/**
+ * Provides a short access to document elements by ID via delegated
+ * property syntax. Received element is not cached and received
+ * directly from the [Document] by calling [Document.getElementById]
+ * function on every property access. Throws an exception if element
+ * is not found or has different type
+ *
+ * To access an element with `theId` ID use the following property declaration
```

Comment:
is the id case insensitive? Otherwise `theId` element won't be found by `theID` property

Reply:
pushed the fix, thanks :)